### DISCUSSION TITLE
Is there a way to convert OG plots to new plots suitable for pooling?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#7706] [Q&A] [answered]
Are the old plots totally doomed? Chances of winning in solo mining are now 10^-100.
Can any chia dev explain here why you chose to cut off the old plots from pooling? You actually favored hpool all along... :|

**Comment by SlugFiller** [comment, marked as answer]
A plot is linked to its reward target. This is a given, and is similar to how PoW works in other crypto. When you provide a "proof" that you mined or farmed a block, you have to broadcast it to the rest of the network. But you need to make sure that someone doesn't steal it and pretend it's THEIR proof. The way it's done is by having your address linked to the proof. In Chia, because the proof comes from the plots, the plots are linked to the farmer's reward address.

In theory, what it should have meant for pools is that you'd have to re-plot every time you join a new pool, since the reward address would have been the pool's. The Chia developers worked very hard to invent the "pool NFT", which acts as an intermediary, in that it is the target for the plot rewards, but also itself is able to point at a pool in a public on-chain way that allows pool managers to ensure you won't re-target it AFTER finding a proof in one of your plots.

As an added bonus, they allowed the plot NFT to point at your own private wallet, to self-pool. This was done to allow you to not just switch between pools, but also switch between solo mining, and pool mining. There's also the ability to have multiple pool NFTs. While wasteful, you could have an NFT for each plot, so you can have fine-grained control as to how much of your plots mine towards which pools, or solo, without having to re-plot anything.

All of this was done to allow moving between farming methods without re-plotting. But, there was no way to have it before it was even developed. The OG plots can only ever point at your own farm. If there was a way to convert a proof from one target to another that's more efficient than re-plotting, then that way could also be used to "steal" someone else's proof after they mined, taking their earnings.

In other words, if "converting" was possible, Chia couldn't work.
